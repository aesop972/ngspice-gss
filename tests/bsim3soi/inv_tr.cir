Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)

.include ./nmos4p0.mod
.include ./pmos4p0.mod
.option TEMP=27C

Vpower VD 0 1.5
Vgnd VS 0 0

Vgate   Gate   VS PULSE(0v 1.5v 100ps 50ps 50ps 200ps 500ps)

MN0 VS Gate Out VS N1 W=10u L=0.18u
MP0 VD Gate Out VS P1 W=20u L=0.18u

.tran 0.01n 600ps
.print tran v(gate) v(out)
.END
