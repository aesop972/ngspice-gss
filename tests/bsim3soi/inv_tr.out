Note: can't find init file.
Warning: vgate: no DC value, transient time 0 value used

Circuit: Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)

Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Circuit: Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
Date: Wed Nov 30 11:36:06  2005


Mx  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
Transient Analysis  Wed Nov 30 11:36:06  2005
--------------------------------------------------------------------------------
Index   time            v(gate)         v(out)          
--------------------------------------------------------------------------------
0	0.000000e+00	0.000000e+00	1.499923e+00	
1	1.000000e-13	0.000000e+00	1.499926e+00	
2	2.000000e-13	0.000000e+00	1.499929e+00	
3	4.000000e-13	0.000000e+00	1.499934e+00	
4	8.000000e-13	0.000000e+00	1.499939e+00	
5	1.600000e-12	0.000000e+00	1.499939e+00	
6	3.200000e-12	0.000000e+00	1.499936e+00	
7	6.400000e-12	0.000000e+00	1.499931e+00	
8	1.280000e-11	0.000000e+00	1.499927e+00	
9	2.480000e-11	0.000000e+00	1.499925e+00	
10	3.680000e-11	0.000000e+00	1.499924e+00	
11	4.880000e-11	0.000000e+00	1.499924e+00	
12	6.080000e-11	0.000000e+00	1.499925e+00	
13	7.280000e-11	0.000000e+00	1.499925e+00	
14	8.480000e-11	0.000000e+00	1.499925e+00	
15	9.680000e-11	0.000000e+00	1.499925e+00	
16	1.000000e-10	0.000000e+00	1.499926e+00	
17	1.012000e-10	3.600000e-02	1.518825e+00	
18	1.036000e-10	1.080000e-01	1.545981e+00	
19	1.084000e-10	2.520000e-01	1.569785e+00	
20	1.165982e-10	4.979471e-01	1.552763e+00	
21	1.248647e-10	7.459399e-01	1.457568e+00	
22	1.368647e-10	1.105940e+00	3.487907e-01	
23	1.383647e-10	1.150940e+00	2.568078e-01	
24	1.413647e-10	1.240940e+00	1.494958e-01	
25	1.473647e-10	1.420940e+00	7.166569e-02	
26	1.500000e-10	1.500000e+00	6.172928e-02	
27	1.508639e-10	1.500000e+00	4.411292e-02	
28	1.525917e-10	1.500000e+00	1.899369e-02	
29	1.560474e-10	1.500000e+00	2.395503e-03	
30	1.629587e-10	1.500000e+00	8.751862e-05	
31	1.749587e-10	1.500000e+00	7.576483e-04	
32	1.869587e-10	1.500000e+00	4.320918e-04	
33	1.989587e-10	1.500000e+00	5.877679e-04	
34	2.109587e-10	1.500000e+00	5.110274e-04	
35	2.229587e-10	1.500000e+00	5.467507e-04	
36	2.349587e-10	1.500000e+00	5.279699e-04	
37	2.469587e-10	1.500000e+00	5.356138e-04	
38	2.589587e-10	1.500000e+00	5.304715e-04	
39	2.709587e-10	1.500000e+00	5.317189e-04	
40	2.829587e-10	1.500000e+00	5.294634e-04	
41	2.949587e-10	1.500000e+00	5.291410e-04	
42	3.069587e-10	1.500000e+00	5.278195e-04	
43	3.189587e-10	1.500000e+00	5.270523e-04	
44	3.309587e-10	1.500000e+00	5.257239e-04	
45	3.429587e-10	1.500000e+00	5.251964e-04	
46	3.500000e-10	1.500000e+00	5.243470e-04	
47	3.512000e-10	1.464000e+00	-2.443080e-02	
48	3.536000e-10	1.392000e+00	-5.445157e-02	
49	3.584000e-10	1.248000e+00	-5.622750e-02	
50	3.663851e-10	1.008448e+00	-1.329095e-02	
51	3.743531e-10	7.694084e-01	9.220049e-02	
52	3.863531e-10	4.094084e-01	1.185404e+00	
53	3.878531e-10	3.644084e-01	1.250883e+00	
54	3.908531e-10	2.744084e-01	1.336683e+00	

Index   time            v(gate)         v(out)          
--------------------------------------------------------------------------------
55	3.968531e-10	9.440838e-02	1.411057e+00	
56	4.000000e-10	0.000000e+00	1.422305e+00	
57	4.012000e-10	0.000000e+00	1.445294e+00	
58	4.036000e-10	0.000000e+00	1.476839e+00	
59	4.084000e-10	0.000000e+00	1.496998e+00	
60	4.180000e-10	0.000000e+00	1.500554e+00	
61	4.300000e-10	0.000000e+00	1.499696e+00	
62	4.420000e-10	0.000000e+00	1.499977e+00	
63	4.540000e-10	0.000000e+00	1.499855e+00	
64	4.660000e-10	0.000000e+00	1.499947e+00	
65	4.780000e-10	0.000000e+00	1.499946e+00	
66	4.900000e-10	0.000000e+00	1.499917e+00	
67	5.020000e-10	0.000000e+00	1.499902e+00	
68	5.140000e-10	0.000000e+00	1.499922e+00	
69	5.260000e-10	0.000000e+00	1.499937e+00	
70	5.380000e-10	0.000000e+00	1.499921e+00	
71	5.500000e-10	0.000000e+00	1.499907e+00	
72	5.620000e-10	0.000000e+00	1.499922e+00	
73	5.740000e-10	0.000000e+00	1.499934e+00	
74	5.860000e-10	0.000000e+00	1.499921e+00	
75	5.980000e-10	0.000000e+00	1.499910e+00	
76	6.000000e-10	0.000000e+00	1.499916e+00	

CPU time since last call: 0.280 seconds.

Total CPU time: 0.280 seconds.

Current dynamic memory usage = 344064,
Dynamic memory limit = -18823216.

